Makefile

```diff
@@ -161,7 +161,7 @@ test-plugins:
 
 test-relay-integration:
 	@echo "--> Running Relay integration tests"
-	pytest tests/relay_integration -vv
+	pytest tests/relay_integration -vv --cov . --cov-report="xml:.artifacts/relay.coverage.xml" --junit-xml=".artifacts/relay.junit.xml"
```

Comment:
Can I assume this is intentional?

Reply:
Yeah I initially refactored these paths and noticed relay wasn't outputting coverage stats.